docs/source/gcp/examples/kms-cryptokey-version.rst

```diff
@@ -0,0 +1,21 @@
+Key Management System - Audit Crypto Key Version destruction
+============================================================
+Custodian can audit and notify if a Version for a KMS Cryptographic Key has been destroyed. Note that the ``notify`` action requires a Pub/Sub topic to be configured.
+
+.. code-block:: yaml
+
+    policies:
+        - name: gcp-kms-cryptokey-version-audit-destruction
```

Comment:
this example doesnt seem useful, i'd suggest removing it. These generic policies of form notify on api call X for resource Y should be avoided.